src/Cart/Helpers/CartConditionHelper.php

```diff
@@ -50,7 +50,8 @@ protected function processActionValue($action, $total)
             $value = (float)$this->cleanValue($actionValue);
         }
 
-        $this->calculatedValue += $value;
+        $precision = app('currency')->getDefault() ? app('currency')->getDefault()->decimal_position : 2;
+        $this->calculatedValue += round($value, $precision);
```

Comment:
Let's pass the currency precision to the Cart library/CartCondition as config or property.

Should we also implement precision when calculating any money values within the Cart library.. For example, when calculating the cart item total?